src/ontology/uberon-edit.obo

```diff
@@ -214663,10 +214663,13 @@ property_value: provenance_notes "This class was sourced from an external ontolo
 
 [Term]
 id: UBERON:3011045
-name: gasserian ganglion
+name: obsolete gasserian ganglion
 synonym: "gasserian ganglia" RELATED [UBERON:3011045]
 is_a: UBERON:0000045 ! ganglion
 property_value: provenance_notes "This class was sourced from an external ontology (amphibian_anatomy). Its definitions, naming conventions and relationships may need to be checked for compatibility with uberon" xsd:string {source="http://purl.obolibrary.org/obo/aao.owl"}
+comment: Obsoleted as gasserian ganglion is an exact synonym to trigeminal ganglion per Wikipedia
+is_obsolete: true
+consider: UBERON:0001675
```

Comment:
Didnt you say they are exact replacements? You should use `replaced_by` then.

Reply:
it is synoyn: EXACT. I will use replaced_by instead of consider